ui-admin/src/study/participants/participantList/facets/EnrolleeSearchFacets.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+
+import { Accordion } from 'react-bootstrap'
+import _cloneDeep from 'lodash/cloneDeep'
+import {
+  Facet,
+  FacetValue, IntRangeFacetValue,
+  newFacetValue, StableIdStringArrayFacetValue,
+  StringFacetValue
+} from 'api/enrolleeSearch'
+import IntRangeFacetView from './IntRangeFacetView'
+import StableIdStringFacetView from './StableIdStringFacetView'
+import StringFacetView from './StringFacetView'
+
+/**
+ * Renders a list of facets in an accordion.  Takes an array of facet values -- this array should represent only
+ * those facets that have user-specified non-default values.
+ */
+export default function EnrolleeSearchFacets({ facets, facetValues, updateFacetValues }:
+    {facets: Facet[], facetValues: FacetValue[], updateFacetValues: (values: FacetValue[]) => void}) {
```

Comment:
Style nit: to make function signatures like this less noisy, I've been using this style:

```suggestion
type EnrolleeSearchFacetsProps = {
  facets: Facet[]
  facetValues: FacetValue[]
  updateFacetValues: (values: FacetValue[]) => void
}

/**
 * Renders a list of facets in an accordion.  Takes an array of facet values -- this array should represent only
 * those facets that have user-specified non-default values.
 */
export default function EnrolleeSearchFacets(props: EnrolleeSearchFacetsProps) {
  const { facets, facetValues, updateFacetValues } = props
```